.azure/azure-build-pipeline.yml

```diff
@@ -0,0 +1,44 @@
+trigger:
+- main
+
+pr: 
+- main
+
+stages:
+  - stage: 'Test'
+    jobs:
+    - job: RunTests
+      pool:
+        vmImage: ubuntu-20.04
+      strategy:
+        matrix:
+          Python3.6:
+            python.version: '3.6'
+          Python3.7:
+            python.version: '3.7'
+          Python3.8:
+            python.version: '3.8'
+          Python3.9:
+            python.version: '3.9'
+        maxParallel: 3
+      steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: $(python.version)
+        displayName: 'Setup Python'
+      - script: |
+          pip install virtualenv
+          virtualenv .venv
+        displayName: 'Create Venv'
+      - script: |
+          source .venv/bin/activate 
+          ./run setup
+        displayName: 'Setup development environment'
+      - script: | 
+          source .venv/bin/activate 
+          ./run check -f
+        displayName: 'Check formatting'
+      - script: |   
+          source .venv/bin/activate 
+          ./run test
```

Comment:
tests include system test which takes long time.
Could we execute only unit tests?
```
./run test all unit  # execute unit tests only
```

Reply:
Done, will test everything in weekly build pipeline